tmp/build/library-sync.sh

```diff
@@ -29,6 +34,6 @@ git add operator
 rm t.tar
 rm -rf library-master
 
-echo "REMEMBER TO PATCH THE DATAGRID TEMPLATES TO apps/v1 FOR THEIR StatefulSets INSTANCES,"
-echo " and fix openjdk-8-rhel8:1.1 until jboss fixes,"
-echo " and nodejs* templates off of v8"
\ No newline at end of file
+echo "REMEMBER:"
+echo " - fix openjdk-8-rhel8:1.1 until jboss fixes,"
```

Comment:
Not relevant to this PR, but FYI this has been fixed in their `develop` branch, not sure why the library is using a specific tag for openjdk x86 images.

Reply:
they are in the process of switching to the `release`  tag/branch